automation/src/test/scala/org/broadinstitute/dsde/workbench/sam/api/SamApiSpec.scala

```diff
@@ -184,169 +170,20 @@ class SamApiSpec extends AnyFreeSpec with BillingFixtures with Matchers with Sca
       val authToken1: AuthToken = user1.makeAuthToken()
       val authToken2: AuthToken = user2.makeAuthToken()
 
-      withCleanBillingProject(UserPool.chooseProjectOwner, List(user1.email, user2.email)) { project =>
-        val email = WorkbenchEmail(Sam.user.status()(authToken1).get.userInfo.userEmail)
-        val username = email.value.split("@").head
-        val userId = Sam.user.status()(authToken1).get.userInfo.userSubjectId
-
-        val petSAEmail = Sam.user.petServiceAccountEmail(project)(authToken1)
-
-        val proxyGroup_1 = Sam.user.proxyGroup(petSAEmail.value)(authToken1)
-        val proxyGroup_2 = Sam.user.proxyGroup(petSAEmail.value)(authToken2)
-
-        val expectedProxyEmail = s"$userId@${gcsConfig.appsDomain}"
-
-        proxyGroup_1.value should endWith(expectedProxyEmail)
-        proxyGroup_2.value should endWith(expectedProxyEmail)
-      }
-    }
-
-    "should furnish a new service account key and cache it for further retrievals" in {
-      val user = UserPool.chooseStudent
-
-      withCleanBillingProject(UserPool.chooseProjectOwner, List(user.email)) { project =>
-        withCleanUp {
-          val key1 = Sam.user.petServiceAccountKey(project)(user.makeAuthToken())
-          val key2 = Sam.user.petServiceAccountKey(project)(user.makeAuthToken())
-
-          key1 shouldBe key2
-
-          register cleanUp Sam.user.deletePetServiceAccountKey(project, getFieldFromJson(key1, "private_key_id"))(user.makeAuthToken())
-        }
-      }
-    }
-
-    "should furnish a new service account key after deleting a cached key" in {
```

Comment:
covered by unit test [create a service account key and return the same key when called again](https://github.com/broadinstitute/sam/blob/ef8307384ce48589dffa5d995e3ba66e03d9dfea/src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionSpec.scala#L726)